examples/ABAQUS_IceCreamCone.control

```diff
@@ -0,0 +1,78 @@
+%
```

Comment:
I'd probably rename this file to `IceCreamCone_Abaqus.control` since I consider the output format a secondary information to the mesh shape.

Reply:
Sure, that makes sense.